moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -34,9 +34,10 @@
 
 /* Author: Mrinal Kalakrishnan */
 
-#include <ros/ros.h>
 #include <chomp_motion_planner/chomp_trajectory.h>
 
+#include "rclcpp/rclcpp.hpp"
```

Comment:
Also here, if there is no apparent change with this header, it can probably just be removed